### DISCUSSION TITLE
How to enter the dynamic route parameters in dev mode

### DISCUSSION THREAD

**Discussion by Complexlity** [wevm/frog-archived#73] [Q&A]
![image](https://github.com/wevm/frog/assets/105590967/68ee57bf-fe74-4769-886f-4e6e77e50b1e)

I created a dynamic route. But from the dev url dropdown, I only see the option to click the url with the placeholder `:id`
I think there should be a way to actually enter the `:id` in some kind of input so it uses that to fetch instead

**Comment by Complexlity** [comment]
Oh I found out I could enter it in the url like so:
```
http://localhost:3000/api/item/33x1aw2/dev
```

**Comment by tmm** [comment]
Added improved support for this in frog@0.7
